web/static/help/Messaging_es.md

```diff
@@ -0,0 +1 @@
+../../../doc/help/Messaging_es.md
```

Comment:
Something we'll need to fix with the build un-related to loc.  For now we should duplicate the file.